// react-useanimations/stories/generated.stories.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.All = exports["default"] = void 0;

var _react = _interopRequireDefault(require("react"));

var _activity = _interopRequireDefault(require("../lib/activity"));

var _airplay = _interopRequireDefault(require("../lib/airplay"));

var _alertCircle = _interopRequireDefault(require("../lib/alertCircle"));

var _alertOctagon = _interopRequireDefault(require("../lib/alertOctagon"));

var _alertTriangle = _interopRequireDefault(require("../lib/alertTriangle"));

var _archive = _interopRequireDefault(require("../lib/archive"));

var _arrowDown = _interopRequireDefault(require("../lib/arrowDown"));

var _arrowDownCircle = _interopRequireDefault(require("../lib/arrowDownCircle"));

var _arrowLeftCircle = _interopRequireDefault(require("../lib/arrowLeftCircle"));

var _arrowRightCircle = _interopRequireDefault(require("../lib/arrowRightCircle"));

var _arrowUp = _interopRequireDefault(require("../lib/arrowUp"));

var _arrowUpCircle = _interopRequireDefault(require("../lib/arrowUpCircle"));

var _bookmark = _interopRequireDefault(require("../lib/bookmark"));

var _calendar = _interopRequireDefault(require("../lib/calendar"));

var _checkBox = _interopRequireDefault(require("../lib/checkBox"));

var _codepen = _interopRequireDefault(require("../lib/codepen"));

var _copy = _interopRequireDefault(require("../lib/copy"));

var _download = _interopRequireDefault(require("../lib/download"));

var _edit = _interopRequireDefault(require("../lib/edit"));

var _explore = _interopRequireDefault(require("../lib/explore"));

var _facebook = _interopRequireDefault(require("../lib/facebook"));

var _github = _interopRequireDefault(require("../lib/github"));

var _heart = _interopRequireDefault(require("../lib/heart"));

var _infinity = _interopRequireDefault(require("../lib/infinity"));

var _info = _interopRequireDefault(require("../lib/info"));

var _instagram = _interopRequireDefault(require("../lib/instagram"));

var _linkedin = _interopRequireDefault(require("../lib/linkedin"));

var _loading = _interopRequireDefault(require("../lib/loading"));

var _loading2 = _interopRequireDefault(require("../lib/loading2"));

var _loading3 = _interopRequireDefault(require("../lib/loading3"));

var _lock = _interopRequireDefault(require("../lib/lock"));

var _mail = _interopRequireDefault(require("../lib/mail"));

var _maximizeMinimize = _interopRequireDefault(require("../lib/maximizeMinimize"));

var _maximizeMinimize2 = _interopRequireDefault(require("../lib/maximizeMinimize2"));

var _menu = _interopRequireDefault(require("../lib/menu"));

var _menu2 = _interopRequireDefault(require("../lib/menu2"));

var _menu3 = _interopRequireDefault(require("../lib/menu3"));

var _menu4 = _interopRequireDefault(require("../lib/menu4"));

var _microphone = _interopRequireDefault(require("../lib/microphone"));

var _microphone2 = _interopRequireDefault(require("../lib/microphone2"));

var _notification = _interopRequireDefault(require("../lib/notification"));

var _notification2 = _interopRequireDefault(require("../lib/notification2"));

var _playPause = _interopRequireDefault(require("../lib/playPause"));

var _playPauseCircle = _interopRequireDefault(require("../lib/playPauseCircle"));

var _plusToX = _interopRequireDefault(require("../lib/plusToX"));

var _pocket = _interopRequireDefault(require("../lib/pocket"));

var _radioButton = _interopRequireDefault(require("../lib/radioButton"));

var _scrollDown = _interopRequireDefault(require("../lib/scrollDown"));

var _searchToX = _interopRequireDefault(require("../lib/searchToX"));

var _settings = _interopRequireDefault(require("../lib/settings"));

var _settings2 = _interopRequireDefault(require("../lib/settings2"));

var _share = _interopRequireDefault(require("../lib/share"));

var _skipBack = _interopRequireDefault(require("../lib/skipBack"));

var _skipForward = _interopRequireDefault(require("../lib/skipForward"));

var _star = _interopRequireDefault(require("../lib/star"));

var _thumbUp = _interopRequireDefault(require("../lib/thumbUp"));

var _toggle = _interopRequireDefault(require("../lib/toggle"));

var _trash = _interopRequireDefault(require("../lib/trash"));

var _trash2 = _interopRequireDefault(require("../lib/trash2"));

var _twitter = _interopRequireDefault(require("../lib/twitter"));

var _userMinus = _interopRequireDefault(require("../lib/userMinus"));

var _userPlus = _interopRequireDefault(require("../lib/userPlus"));

var _userX = _interopRequireDefault(require("../lib/userX"));

var _video = _interopRequireDefault(require("../lib/video"));

var _video2 = _interopRequireDefault(require("../lib/video2"));

var _visibility = _interopRequireDefault(require("../lib/visibility"));

var _visibility2 = _interopRequireDefault(require("../lib/visibility2"));

var _volume = _interopRequireDefault(require("../lib/volume"));

var _youtube = _interopRequireDefault(require("../lib/youtube"));

var _youtube2 = _interopRequireDefault(require("../lib/youtube2"));

var _ = _interopRequireDefault(require(".."));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var _default = {
  title: 'Animations',
  component: _["default"]
};
exports["default"] = _default;

var All = function All() {
  return /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      display: 'flex',
      flexWrap: 'wrap'
    }
  }, /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "activity"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _activity["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "airplay"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _airplay["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "alertCircle"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _alertCircle["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "alertOctagon"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _alertOctagon["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "alertTriangle"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _alertTriangle["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "archive"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _archive["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "arrowDown"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _arrowDown["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "arrowDownCircle"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _arrowDownCircle["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "arrowLeftCircle"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _arrowLeftCircle["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "arrowRightCircle"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _arrowRightCircle["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "arrowUp"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _arrowUp["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "arrowUpCircle"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _arrowUpCircle["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "bookmark"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _bookmark["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "calendar"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _calendar["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "checkBox"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _checkBox["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "codepen"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _codepen["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "copy"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _copy["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "download"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _download["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "edit"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _edit["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "explore"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _explore["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "facebook"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _facebook["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "github"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _github["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "heart"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _heart["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "infinity"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _infinity["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "info"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _info["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "instagram"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _instagram["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "linkedin"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _linkedin["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "loading"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _loading["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "loading2"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _loading2["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "loading3"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _loading3["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "lock"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _lock["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "mail"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _mail["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "maximizeMinimize"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _maximizeMinimize["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "maximizeMinimize2"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _maximizeMinimize2["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "menu"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _menu["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "menu2"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _menu2["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "menu3"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _menu3["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "menu4"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _menu4["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "microphone"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _microphone["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "microphone2"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _microphone2["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "notification"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _notification["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "notification2"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _notification2["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "playPause"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _playPause["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "playPauseCircle"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _playPauseCircle["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "plusToX"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _plusToX["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "pocket"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _pocket["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "radioButton"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _radioButton["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "scrollDown"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _scrollDown["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "searchToX"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _searchToX["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "settings"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _settings["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "settings2"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _settings2["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "share"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _share["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "skipBack"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _skipBack["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "skipForward"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _skipForward["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "star"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _star["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "thumbUp"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _thumbUp["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "toggle"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _toggle["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "trash"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _trash["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "trash2"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _trash2["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "twitter"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _twitter["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "userMinus"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _userMinus["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "userPlus"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _userPlus["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "userX"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _userX["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "video"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _video["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "video2"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _video2["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "visibility"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _visibility["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "visibility2"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _visibility2["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "volume"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _volume["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "youtube"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _youtube["default"]
  })), /*#__PURE__*/_react["default"].createElement("div", {
    style: {
      padding: '20px'
    }
  }, /*#__PURE__*/_react["default"].createElement("span", null, "youtube2"), /*#__PURE__*/_react["default"].createElement(_["default"], {
    size: 40,
    wrapperStyle: {
      marginTop: '5px'
    },
    animation: _youtube2["default"]
  })));
};

exports.All = All;
